docs/identities/sign-in/identifier-first-authentication.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: identifier-first-authentication
+title: Identifier First Authentication
+sidebar_label: Identifier First Authentication
+---
+
+Identifier First Authentication streamlines the login process by first requesting the user's identifier (such as email or
+username) before prompting for a password or other authentication methods. This approach enhances user experience and security by
+tailoring authentication steps based on the user's context.
+
+This guide explains how to enable and use Identifier First Authentication in Ory Network (Ory Identities) and self-hosted Ory
+Kratos.
+
+## Ory Network
+
+To enable it:
+
+1. Log in to the [Ory Console](https://console.ory.sh).
+2. Select your project.
+3. Navigate to [Project Settings > Advanced](https://console.ory.sh/projects/current/settings/advanced).
+4. Enable "New Account Experience".
+5. Click "Save Changes".
+6. Test the flow in your application by navigating to the login page.
+
+To disable this feature,follow the same steps and disable "New Account Experience".
+
+## Self-hosted Ory Kratos
```

Comment:
not sure how I feel about mixing in self-hosted here, but I also have no better idea now...